graveler/committed/manager.go

```diff
@@ -26,6 +27,24 @@ func (c *committedManager) List(ctx context.Context, ns graveler.StorageNamespac
 	panic("implement me")
 }
 
+func (c *committedManager) WriteMetaRange(ctx context.Context, ns graveler.StorageNamespace, it graveler.ValueIterator) (*graveler.MetaRangeID, error) {
+	writer := c.metaRangeManager.NewWriter(ns)
+	for it.Next() {
+		if err := it.Err(); err != nil {
+			return nil, fmt.Errorf("getting value from iterator: %w", err)
+		}
+		if err := writer.WriteRecord(*it.Value()); err != nil {
+			return nil, fmt.Errorf("writing record: %w", err)
```

Comment:
optional: add %s with the key

Reply:
good point - I guess we will have to address a wider solution to trace a request which will include this information.